www/docusaurus.config.js

```diff
@@ -25,7 +25,7 @@ const config = {
   tagline: 'Move Fast and Break Nothing.\nEnd-to-end typesafe APIs made easy.',
   url: 'https://trpc.io',
   baseUrl: '/',
-  onBrokenLinks: 'ignore',
+  onBrokenLinks: 'throw',
```

Comment:
I'm guessing you disable this due to typedoc causing it to throw?

Reply:
Returned back the previous code.